CMakeLists.txt

```diff
@@ -140,15 +140,22 @@ if(BUILD_TESTS)
     ${CMAKE_CURRENT_SOURCE_DIR}/src/enclave/test/http.cpp)
   target_link_libraries(http_test PRIVATE http_parser.host)
 
-  if(NOT PBFT)
-    add_unit_test(frontend_test
+  add_unit_test(frontend_test
       ${CMAKE_CURRENT_SOURCE_DIR}/src/node/rpc/test/frontend_test.cpp)
     target_link_libraries(frontend_test PRIVATE
       ${CMAKE_THREAD_LIBS_INIT}
       evercrypt.host
       lua.host
-      secp256k1.host)
+      secp256k1.host
+    )
+    if (PBFT)
+      target_link_libraries(frontend_test PRIVATE
+        -Wl,--allow-multiple-definition #TODO(#important): This is unfortunate
+        libcommontest
```

Comment:
What's in libcommontest that we need in frontend_test?

Reply:
everything ITimer related

Reply:
Just noticed it breaks if we are running with sgx. I will change it to link to `libbyz.host` if we are in virtual or `libbyz.enclave` if we are in sgx mode